app/src/main/AndroidManifest.xml

```diff
@@ -26,6 +26,8 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <meta-data android:name="firebaseDebugging"
```

Comment:
Can this be set from here: https://github.com/covid19risk/covidwatch-android-en/blob/dc6dcaabacc1f583e5ad186178b087353ef57b1e/app/build.gradle#L24-L35